lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -298,8 +298,24 @@ TypeSystemSwiftTypeRef::GetClangTypeNode(CompilerType clang_type,
   llvm::StringRef swift_name;
   llvm::StringRef module_name = swift::MANGLING_MODULE_OBJC;
   CompilerType pointee;
-  if (clang_type.IsPointerType(&pointee))
+  if (clang_type.IsPointerType(&pointee)) {
     clang_type = pointee;
+    if (clang_type.IsVoidType()) {
+      // Sugar (void *) as "UnsafeMutableRawPointer?".
```

Comment:
why optional `UnsafeMutableRawPointer` vs non-optional?

Reply:
should this code check for `const void *` and map that to `UnsafeRawPointer`?

Reply:
I'm matching what SwiftASTContext produces. I assume it's because the pointer is nullable?